spec/ad-engine/listeners/scroll-listener.spec.ts

```
@@ -0,0 +1,60 @@
+import { ScrollListener } from '@wikia/ad-engine';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+describe('ScrollListener', () => {
+	let scrollListener: ScrollListener;
+	let getElementByIdStub;
+	let adStack: any[];
```

what about `OldLazyQueue<AdStackPayload>`? It would silence errors.

---

Not really - `OldLazyQueue` interface includes `start` method, which does not exist on an Array.